src/test/jtx/Env_test.cpp

```diff
@@ -49,13 +49,6 @@ class Env_test : public beast::unit_test::suite
     testAccount()
     {
         using namespace jtx;
-        {
-            Account a;
-            Account b(a);
-            a = b;
-            a = std::move(b);
-            Account c(std::move(a));
-        }
         Account("alice");
         Account("alice", KeyType::secp256k1);
         Account("alice", KeyType::ed25519);
```

Comment:
I'm not sure what these things test here... that these constructors are callable? If so, then these can be converted to `static_assert`.

Reply:
Yes, I think the portion of the code I have removed tests copy and move constructors' callability.